storage/innobase/xtrabackup/doc/source/backup_scenarios/incremental_backup.rst

```diff
@@ -206,6 +206,12 @@ first incremental backup. If you restore the files from
 :file:`/data/backups/base`, you should see the state of the database as of the
 first incremental backup.
 
+.. warning::
+
+   Do not run :option:`xtrabackup --prepare` with the same incremental backup
```

Comment:
Backup doesn't become corrupt... Just using the same incremental backup dir to prepare two copies of backup is not supported